bundler/doc/development/PULL_REQUESTS.md

```diff
@@ -10,7 +10,7 @@ Before you submit a pull request, please remember to do the following:
 
 Make sure the code formatting and styling adheres to the guidelines. We use RuboCop for this. Lack of formatting adherence will result in automatic GitHub Actions build failures.
 
-      $ bin/rubocop -a
+      $ rake rubocop
```

Comment:
Actually there's `bin/rubocop` at `bundler/` folder in the repo, which is where you usually work when working on Bundler. Should we instead clarify that?

Reply:
Oh, I see! Indeed, this wasn’t very clear to me.

Perhaps by prefixing a `cd …` it would be enough for newcomers to understand?

```suggestion
      $ cd bundler
      $ bin/rubocop -a
```

Reply:
Indeed, that's a good idea! 👍